docs/pages/versions/unversioned/sdk/in-app-purchases.md

```diff
@@ -0,0 +1,304 @@
+
+On iOS, the refresh boolean is ignored and it returns the purchase history in the same way that `connectAsync` does. An important thing to note is that on iOS, Storekit actually creates a new transaction object every time you restore completed transactions, therefore the `purchaseTime` and `orderId` may be inaccurate if it's a restored purchase. If you need the original transaction's information you can use `originalPurchaseTime` and `originalOrderId`, but those will be 0 and an empty string respectively if it is the original transaction.
+
+#### Arguments
+
+- **refresh (_boolean_)** -- A boolean that indicates whether or not you want to make a network request to sync expired/consumed purchases and those on other devices (Android only)
+
+#### Returns
+
+A `Promise` that is resolved with a `QueryResponse` that contains an array of `Purchase` objects.
+
+#### Example
+```javascript
+const { responseCode, results } = await getPurchaseHistoryAsync();
+if (responseCode === ResponseCode.OK) {
+    results.forEach(result => {
+        // Handle purchase history
+    })
+}
+```
+
+### `InAppPurchases.purchaseItemAsync(productId: string, oldItem?: string)`
+
+Initiates the purchase flow to buy the item associated with this productId. This function is void and the result must be handled in the callback that you passed in to `setPurchaseListener`. This will display a prompt to the user which will allow them to either buy the item or cancel the purchase.
+
+#### Arguments
+
+- **productId (_string_)** -- The product ID of the item you want to buy.
+
+- **oldItem (_string_)** -- The product ID of the item you want to replace with this new purchase, typically a subscription such as when a user upgrades from monthly to yearly (Android only).
+
+#### Example
+```javascript
+renderItem(item) {
+    // Render product details with a "Buy" button
+    return (
+        <View key={item.productId}>
+            ...
+            <View style={styles.buttonContainer}>
+                <Button title="Buy" onPress={() => purchaseItemAsync(item.productId)} />
+            </View>
+            ...
+        </View>
+    );
+}
+```
+
+### `InAppPurchases.finishTransactionAsync(purchase: Purchase, consumeItem: boolean)`
+
+Marks a transaction as completed. This must be called on successful purchases only after you have successfully processed the transaction and unlocked the functionality purchased by the user.
+
+On Android, this will either "acknowledge" or "consume" the purchase depending on the value of `consumeItem`. Consuming a purchase allows it to be bought more than once. You cannot buy an item again until it's consumed. Both consuming and acknowledging let Google know that you are done processing the transaction. Acknowledging indicates that this is a one time purchase (e.g. premium upgrade), rather than a consumable. If you do not acknowledge or consume a purchase within three days, the user automatically receives a refund, and Google Play revokes the purchase.
```

Comment:
```suggestion
On Android, this will either "acknowledge" or "consume" the purchase depending on the value of `consumeItem`. Acknowledging indicates that this is a one time purchase (e.g. premium upgrade), whereas consuming a purchase allows it to be bought more than once. You cannot buy an item again until it's consumed. Both consuming and acknowledging let Google know that you are done processing the transaction. If you do not acknowledge or consume a purchase within three days, the user automatically receives a refund, and Google Play revokes the purchase.
```